### ISSUE TITLE
Conditional step always returns expression matching, after one matching request

### ISSUE THREAD

**Issue by joonasroosalung** [buerokratt/Ruuter#121]
Example yml file:

```
checkChatIdParameter:
  switch:
    - condition: ${!!incoming.params.chatId && incoming.params.chatId !== "" && incoming.params.chatId !== null}
      next: returnWithParameter
  next: returnNoParameter

returnWithParameter:
  return: 100
  next: end

returnNoParameter:
  return: "300"
  next: end

returnNotFound:
  return: "error: not found"
  next: end
```

A request with the chatId parameter filled should always respond `{ "response": "100" }`. A request where the parameter is missing or null should always respond `{ "response": "300" }`.

Currently, after a restart requests with a missing or null parameter will respond `{ "response": "300" }`, until a request with a filled parameter after which the respones will be `{ "response": "100" }` regardless if the parameter missing.

**Comment by joonasroosalung** [comment]
Created PR https://github.com/buerokratt/Ruuter/pull/122

**Comment by joonasroosalung** [comment]
Merged PR #122
